db-admin.tf

```diff
@@ -0,0 +1,15 @@
+module "db_admin" {
+  source = "api.nullstone.io/nullstone/aws-pg-db-admin/aws"

```

Comment:
Shouldn't this just be included in this module always? I'm trying to imagine a time when you wouldn't want to have a db without an admin ability.

Reply:
You would always.
I built into a separate module because there are other modules that would want to use this db admin.
The github.com/nullstone-modules/pg-db-admin repo has the same mechanism for postgres regardless of provider (aws, gcp, azure, etc.) or tech (rds, aurora, etc.)

Reply:
Ah, makes sense. Love the pattern.